FlightJournal.Web/Migrations/FlightContext/202105261056416_Add HasTrainingData field to Flight.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace FlightJournal.Web.Migrations.FlightContext
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class AddHasTrainingDatafieldtoFlight : DbMigration
+    {
+        public override void Up()
+        {
+            AddColumn("dbo.Flights", "HasTrainingData", c => c.Boolean(false, false));
```

Comment:
Man kunne overveje i dette step en Custom sql til at preload tidligere data... men er nok enklere som Custom kørsel.

Reply:
Det var nok egentlig en god ide - hvis det felt ikke er gyldig bagudrettet, bliver det noget rod mht præsentation af gamle data (og måske export).

Jeg prøver lige at komme med at bud...

Reply:
update Flights set HasExercise=true where FlightId in (select distinct FlightId from [dbo].[AppliedExercises] where [Grading_GradingId] != 5)

(men skal det ind i Up() og i så fald hvordan ? )